heron/tools/cli/tests/python/parse_unittest.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2017 Twitter. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+''' opts_unittest.py '''
+import unittest2 as unittest
+import mock
+from mock import call, patch, Mock, MagicMock
+import os
+import subprocess
+import sys
+import tempfile
+import heron.tools.cli.src.python.main as main
+import heron.tools.cli.src.python.submit as submit
+import heron.tools.cli.src.python.result as result
+import heron.tools.common.src.python.utils.config as config
+
+class OptsTest(unittest.TestCase):
```

Comment:
`OptsTest` sounds like a test that verifies option parsing logic. Can we call this something more representative like `SubmitTest` or something (which extends a common `ClientCommandTest` or something like that)? We can then have a bunch of tests to represent how different args to submit result in different popens.